apps/fz_http/lib/fz_http/api_tokens.ex

```diff
@@ -0,0 +1,60 @@
+defmodule FzHttp.ApiTokens do
+  @moduledoc """
+  The ApiTokens context.
+  """
+
+  import Ecto.Query, warn: false
+  alias FzHttp.Repo
+
+  alias FzHttp.ApiTokens.ApiToken
+
+  def count_by_user_id(user_id) do
+    Repo.aggregate(from(a in ApiToken, where: a.user_id == ^user_id), :count)
+  end
+
+  def list_api_tokens do
+    Repo.all(ApiToken)
+  end
+
+  def list_api_tokens(user_id) do
+    Repo.all(from a in ApiToken, where: a.user_id == ^user_id)
+  end
+
+  def get_api_token(id), do: Repo.get(ApiToken, id)
+
+  def get_api_token!(id), do: Repo.get!(ApiToken, id)
+
+  def get_unexpired_api_token(api_token_id) do
+    Repo.one(
+      from a in ApiToken,
+        where: a.id == ^api_token_id and fragment("? > now()", a.expires_at)
+    )
+  end
+
+  def new_api_token(attrs \\ %{}) do
+    ApiToken.create_changeset(attrs)
+  end
+
+  def create_user_api_token(%FzHttp.Users.User{} = user, attrs) do
+    changeset =
+      attrs
+      |> Enum.into(%{user_id: user.id})
```

Comment:
We can just pass it to the changeset and use `put_assoc` there. This will make the changeset code more explicit like we did here in the context. It looks even nicer because we have a changeset function per action, so their argument is different and the update changeset won't even accept the user anymore as the user can't be changed.

Reply:
Later having attributes like this one will be handy so we can use them for auth checks, eg. if auth subject has permission to manage user tokens.